### PR TITLE
[CORE-2258] Changelog/commit message for update 1

### PR DESCRIPTION
**Pull request by mbaynton**
Allow site-level customization of Drush version, downgrade to D8, and other
fixes. More info in CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Two repositories for release management. Pull requests accepted at https://github.com/pantheon-systems/drupal-project and releases located at https://github.com/pantheon-upstreams/drupal-project. (#4,#5)
+
+
+
+### Changed
+
+- Allow site-level customization of Drush version. (#5)
+
+- Allow sites to downgrade to Drupal 8.8.  (#6)
+- Use optimized autoloader in Test and Live, but not Dev or Multidev (#8)
```
Comment (ari-gold) [inline CHANGELOG.md:20]: Looks like we have empty lines between each item, except for this line? Seems like we can get rid of the empty lines? The markdown renders fine though, so defer to you.

**Review by ari-gold — APPROVED** [review]

**Comment by ari-gold** [comment]
In the commit message, if we linked to Changleog: https://github.com/pantheon-upstreams/drupal-project/blob/master/CHANGELOG.md.   would that be rendered as a link that customers could click on for more info?

**Comment by mbaynton** [comment]
Doesn't seem to be currently supported on the dashboard, I tried a link yesterday https://admin.dashboard.pantheon.io/sites/b909ef77-fb72-45d8-ae00-07659c3055cd#dev/code

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+
+## Pantheon Update #1 - 2021-02-03
+
+### Added
+
+- Add changelog to track Pantheon-specific changes to pantheon-upstreams/pantheon-project. (#10)
+
+- Two repositories for release management. Pull requests accepted at https://github.com/pantheon-systems/drupal-project and releases located at https://github.com/pantheon-upstreams/drupal-project. (#4,#5)
```
Comment (mbaynton) [inline CHANGELOG.md]:
```suggestion
- Add changelog to track Pantheon-specific changes to pantheon-upstreams/pantheon-project. (#10)
- Two repositories for release management. Pull requests accepted at https://github.com/pantheon-systems/drupal-project and releases located at https://github.com/pantheon-upstreams/drupal-project. (#4,#5)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+
+### Changed
+
+- Allow site-level customization of Drush version. (#5)
+
```
Comment (mbaynton) [inline CHANGELOG.md]:
```suggestion
```

**Review by mbaynton — COMMENTED** [review]
Clean up line spacing

File: CHANGELOG.md (hunk 2 above)
Comment (mbaynton) [reply CHANGELOG.md:13]: Clean up line spacing

**Pull request merged**